.github/codecov.yml

```diff
@@ -0,0 +1,2 @@
+ignore:
+  - 'scripts/**'
```

Comment:
Codecov does not allow to merge this PR otherwise
https://docs.codecov.com/docs/ignoring-paths